portfolio/src/main/webapp/style.css

```
@@ -1,9 +1,117 @@
-#content {
-  margin-left: auto;
-  margin-right: auto;
-  width: 650px;
+html {
+  scroll-behavior: smooth;
+  overflow-x: hidden;
+  background: #0a192f;
+  font-family: 'Ubuntu', sans-serif;
+
+}
+.header{
```

Add space between the selector and the opening `{`
```suggestion
.header {
```